tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js

```diff
@@ -0,0 +1,14 @@
+export function sumFibs(num) {
+  let first = 1;
+  let second = 1;
```

Comment:
These two are semantically more like:

```suggestion
  let previous = 1;
  let next = 1;
```